docs/start/overview.md

```diff
@@ -8,21 +8,100 @@ activePill: overview
+"Service B" could be the next version of "Service A".  Updating the weight on Service B would smoothly transition the 
+load to the new deploy, giving it ample time to scale up if needed.  This is ideal for graceful blue/green deployments.
+
+
+## Shield: The Proxy
+
+Shield takes advantage of it's position as a proxy to help guard the upstream services during times of trouble.  Each 
+upstream service is guarded by a [Circuit Breaker](http://doc.akka.io/docs/akka/2.4.4/common/circuitbreaker.html) so that
+Shield will automatically re-route or shed traffic if the upstream service is experiencing consistent failures or slowdowns.  
+Additionally, each endpoint on the upstream service has it's own dedicated Circuit Breaker so that one endpoint that 
+relies on a flaky resource does not bring down the whole service.  
+
+In the example below, **Service B** has experienced a failure and it's circuit breaker has opened, resulting in Shield 
+immediately returning a failure response.
+
+<img src="{{ site.baseurl }}/assets/images/proxying.gif" class="img-responsive">
+
+While the lingua franca of the web is HTTP, it's very common to use a different method for communicating between the 
+services in your architecture.  Shield supports each upstream service having a different transport mechanism.  The 
+incoming request will be translated and sent across the wire in the dialect that the service supports.  Shield currently
+handles HTTP, HTTPS, and AWS Lambda, with more to come in the near future.
+
+Shield will also handle the compression of the requests and responses, inflating or deflating them based off the 
+preferences of the clients and the services.
+
+
+## Shield: The Middleware Engine
+
+Under the hood, Shield uses [Akka's](http://akka.io) Actor model to power request and response middleware.  These 
+middleware can perform common operations like caching, rate limiting, and authentication.  These middleware are all 
+driven by Shield configuration, including the ability to set timeouts that the middleware must respond by.  If the 
+middleware actors does not respond in time, the request processor will auto advance to the next middleware.
+
+<img src="{{ site.baseurl }}/assets/images/middleware.gif" class="img-responsive">
+<img src="{{ site.baseurl }}/assets/images/actor_legend.png" class="img-responsive">
+
+Listeners are similar to middleware, but they receive the request and response pair after the response has already been
+sent to the client.  This allows listeners to asynchronously do their processing without impacting the client experience.
+Common use cases here include logging and analytics.  Like their middleware brethren, listeners are controlled via 
+configuration.
+
+
+## Shield: Production Ready
+
+Shield has been powering RetailMeNot's API since 2016 and has proven to be stable and fast across billions of requests.
```

Comment:
Maybe include the month, so it's apparent that Shield's been in front of PROD for over a year :D

(Unless it hasn't been a year. In that case don't include the month, lol.)